modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -461,22 +459,85 @@ inline Ort::Value createTensor(const Ort::MemoryInfo& memory_info,
 }
 
 inline Ort::Value createTensor(const Ort::MemoryInfo& memory_info,
-                               const cv::gimpl::onnx::TensorInfo& tensor_params,
-                               const cv::Mat& data) {
+const cv::gimpl::onnx::TensorInfo& tensor_params, const cv::Mat& data) {
     GAPI_Assert(data.isContinuous ());
     switch (tensor_params.type) {
-    case ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8:
-        return createTensor<uint8_t>(memory_info, tensor_params, data);
-    case ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT:
-        return createTensor<float>(memory_info, tensor_params, data);
-    case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32:
-        return createTensor<int32_t>(memory_info, tensor_params, data);
-    default:
+        case ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8:
+            return createTensor<uint8_t>(memory_info, tensor_params, data);
+        case ONNX_TENSOR_ELEMENT_DATA_TYPE_FLOAT:
+            return createTensor<float>(memory_info, tensor_params, data);
+        case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT32:
+            return createTensor<int32_t>(memory_info, tensor_params, data);
+        case ONNX_TENSOR_ELEMENT_DATA_TYPE_INT64:{
+            // creating int64 vector to store the converted data. This jhas the same size as data.
+            std::vector<int64_t> temp(data.total());
```

Comment:
can we get rid of extra vector?

Reply:
currently using this tensor to store the converted tensors when running the convertInt32ToInt64 which will then be used to deep copy it into i64_tensor